components/edit-collective/sections/Tiers.js

```diff
@@ -421,6 +437,7 @@ class Tiers extends React.Component {
       ...tier,
       type: tier.type || this.defaultType,
       invoiceTemplate: tier.data?.invoiceTemplate,
+      singleTicket: tier.data?.singleTicket,
```

Comment:
I don't understand why we retrieve these fields from `data`, looking at GQLV1's schema, aren't they supposed to be exposed on `Tier` directly?

Reply:
We only expose that in GQLV2. I didn't expose that in V1 as we already expose the `data` field; 

https://github.com/opencollective/opencollective-api/blob/ec4a819584abf672a5410dcf006893e0add01cea/server/graphql/v1/types.js#L1486-L1491. 

I thought no need to add additional fields there as the data field is already exposed.

Reply:
I see, thanks for sharing this context